lib/jsdom/living/events/EventTarget-impl.js

```diff
@@ -234,6 +241,48 @@ function invokeEventListeners(listeners, target, eventImpl) {
 
 const wrappedListener = Symbol("inline event listener wrapper");
 
+/**
+ * Normalize the event listeners options argument in order to get always a valid options object
+ * @param   {Object} options         - user defined options
+ * @param   {Array} defaultBoolKeys  - boolean properties that should belong to the options object
+ * @param   {Array} keysBlackList    - options not allowed (optional)
+ * @returns {Object} object containing at least the  "defaultBoolKeys"
+ */
+function normalizeEventHandlerOptions(options, defaultBoolKeys, keysBlackList) {
+  // normalize the options argument in order to get always an object
+  if (options === null || options === undefined) {
+    options = {};
+  } else if (typeof options !== "object") {
+    options = { capture: options };
+  }
+
+  let defaultOptions;
+  let additionalOptions;
+  keysBlackList = keysBlackList || [];
+
+  try {
```

Comment:
It's complicate because tests and specs are a bit weird. Somehow you should be able to [pass custom options values](https://github.com/w3c/web-platform-tests/blob/eae188126a21a593715ba2626cc059144d6f0978/dom/events/EventListenerOptions-capture.html#L54). If i don't get them tests will fail